### DISCUSSION TITLE
Would it be possible to integrate @spotify/flash-list with react-native-gesture-handler?

### DISCUSSION THREAD

**Discussion by eeshankeni** [software-mansion/react-native-gesture-handler#2140] [Ideas]
I've noticed that when using flash-list with react navigation's vertical stack navigator screens, swipe gestures glitch out and don't work correctly.

This problem happens on the stock react native flat list as well. But using the flatlist that react-native-gesture-handler exports works perfectly.

This would be a handy feature to have and I would like to offer a tip if someone is up for the task.

**Comment by disolaterX** [comment]
Any update on this ?
